MenMa WatchKit Extension/InterfaceController.swift

```diff
@@ -58,11 +58,13 @@ class InterfaceController: WKInterfaceController, WCSessionDelegate {
         
         let messageToSend = ["value": "message sent successfully"]
         session.sendMessage(messageToSend, replyHandler: { replyMessage in
-            //            //handle and present the message on screen
-            //            let value = replyMessage["value"] as? Venue
-            //            //            self.messageLabel.setText(value)
+            //handle and present the message on screen
+            let value = replyMessage["value"] as? Venue
```

Comment:
In my local feature/filter_venue build I don't have anything valuable commented out.
`    override func willActivate() {
        // This method is called when watch view controller is about to be visible to user
        super.willActivate()
        
        let messageToSend = ["value": "message sent successfully"]
        session.sendMessage(messageToSend, replyHandler: { replyMessage in
            //handle and present the message on screen
            let value = replyMessage["value"] as? Venue
            //            self.messageLabel.setText(value)
            print("value: \(value)")
            }, errorHandler: {error in
                // catch any errors here

                print(error)
        })
        
    }`

ah, I see the 'convertedVenuesData' that is being passed into session.sendMessageData